django_fido/models.py

```diff
@@ -91,3 +94,16 @@ def attestation(self) -> AttestationObject:
     def attestation(self, value: AttestationObject):
         self.attestation_data = base64.b64encode(value).decode('utf-8')
         self.credential_id_data = base64.b64encode(value.auth_data.credential_data.credential_id).decode('utf-8')
+
+    @property
+    def level(self) -> Optional[AuthLevel]:
+        """Return certification level."""
+        pass
```

Comment:
`Optional[X]` is basically a shortcut for `Union[X, None]`. Therefore, I'd very much prefer to return an explicit `None`, rather than an implicit one.